From e86ccd7f4d4b575646cc335ea73c937176f20c38 Mon Sep 17 00:00:00 2001
From: Daiki Ueno <ueno@gnu.org>
Date: Mon, 02 Jun 2014 21:22:42 +0000
Subject: tests: Ignore the LINGUAS envvar in msgfmt-desktop-2

Reported by Lars Wendler at:
<http://lists.gnu.org/archive/html/bug-gettext/2014-06/msg00002.html>.
* msgfmt-desktop-2: Unset the LINGUAS envvar.
---
diff --git a/gettext-tools/tests/msgfmt-desktop-2 b/gettext-tools/tests/msgfmt-desktop-2
index 69f1f89..da976b0 100755
--- a/gettext-tools/tests/msgfmt-desktop-2
+++ b/gettext-tools/tests/msgfmt-desktop-2
@@ -111,6 +111,8 @@ Keywords[fr]=one;two;thr\;ee;
 Keywords=Keyword1;Keyword2;Key\;word3;
 EOF
 
+unset LINGUAS
+
 # Sanity checks for contradicting options.
 
 ${MSGFMT} --desktop --template=mf.desktop -d po -o mf.desktop.out \
--
cgit v0.9.0.2
